Dockerfile

```diff
@@ -19,9 +19,19 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager
 
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
+FROM gcr.io/distroless/static:nonroot as operator_image
```

Comment:
I am not sure how we can use reference this downstream.. is there an alternative to this image that would work for you?

Reply:
I didn't change this part - it was there from the beginning. I only added "as operator_image". We probably will need to change it to one of the UBIs if required by the downstream process, but this is not related to this PR

Reply:
I see my bad.. I will try and figure how this is handled downstream.